README.md

```diff
@@ -144,8 +144,11 @@ or if the field `test_inputs` does not contain a list, it would print:
 
 ## Changelog
  * **0.3.2**: 
-    - Change `author` and `packaged_by` fields from List[str] to List[Author] with Author consisting of a dictionary {name: '<Full name>', affiliation: '<Affiliation>', orcid: 'optional orcid id'}
-    - Add `type` field to comply with other bioimage.io RDFs. Only valid value is 'model'.
-    - Add optional `version` field (default 0.1.0) to keep track of model changes.
-    - Add optional `id` field as a unique identifier (the use of [doi](doi.org) is encouraged).
-    - Only allow `license` identifier from the [SPDX license list](https://spdx.org/licenses/).
+    - Breaking changes
+      - The RDF file name in a package should be `rdf.yaml` for all the RDF (not `model.yaml`);
+      - Change `authors` and `packaged_by` fields from List[str] to List[Author] with Author consisting of a dictionary `{name: '<Full name>', affiliation: '<Affiliation>', orcid: 'optional orcid id'}`;
```

Comment:
the general RDF still allows `List[str]`...

Reply:
.. but we can focus on the model RDF here, doesn't hurt (I'm already happy with hits ;-))

Reply:
It's a bit too complicated to clearly describe the distinction between general RDF and model RDF here, so I will leave it like this.